### ISSUE TITLE
White lights can't be seen on white background

### ISSUE THREAD

**Issue by Hipshot-zz** [Garux/netradiant-custom#178]
Maybe light entities can have either an inner/outer black outline and the color inside, or some other symbol, because if you run a white background on the 2d pane, you can't see them. The white background is also an option you can choose via one of the theme templates, QE4 and Q3R I think.

The red arrow below points to white lights, the green one can clearly be seen.
https://i.ibb.co/hf6trSL/image.png

Maybe make it like any of these?
https://i.ibb.co/YkQVyTY/image.png

The issue didn't occur in older radiant because in the 2d view, there wasn't any color displayed on the lights, only in the 3d view. (At least 1,5 and earlier).

**Comment by Garux** [comment]
I've noticed this lately too, and surprisingly there were exactly zero complaints regarding this a few years old feature.
Apparently people do not use white background at all. It also doesn't play well with MSAA.

If solving this, i'd prefer something light weight not bloating view/code/perf. E.g.
- use colour from entity definition, if white (but this is somewhat visual inconsistency)
- add 'colorize 2D lights' toggle (one more not so required toggle to bloat the UI)

**Comment by Hipshot-zz** [comment]
Yea, I'm a bit old school here, I do use the white BG and I disabled all sorts of msaa and what not. I suppose people just go with the default which was a grey bg here, but in old radiant was white.

The "least" bloaty UI-thing her might be to add an option inside preferences instead, because if this is a rare issue, people is not going to switch this on and off on the fly anyway. Maybe like this? https://i.ibb.co/wgc0D9L/bild.png

**Comment by Garux** [comment]
In fact people are quite picky about colors. They get 'eyes burnt' with bright color scheme and stuff. Last decade trends of UI are shades of dark grey, especially in editing apps.

Checkbox in prefs requiring restart is least intrusive way to resolve this. Just not as good, as something working out of the box.

**Comment by Hipshot-zz** [comment]
Yea, not sure how it can be good out of the box easily here, it's like, lets say you fix it for 255 255 255 as BG, then what happens if someone uses like, (for some reason) 250 250 250 (almost white)... maybe there could be a range. 

I forgot that this also effects the radius display of course, not only the point entity.

**Comment by Hipshot-zz** [comment]
(The V (Value/Brightness) in the HSV value isn't used to tint the light symbol either btw, so a "dark green" light will be displayed as a regular green).

**Comment by Garux** [comment]
Right. Compiler normalizes colour, so does editor, which also helps display consistency.
